modules/installation-launching-installer.adoc

```diff
@@ -534,6 +534,37 @@ INFO Time elapsed: 36m22s
 * It is recommended that you use Ignition config files within 12 hours after they are generated because the 24-hour certificate rotates from 16 to 22 hours after the cluster is installed. By using the Ignition config files within 12 hours, you can avoid installation failure if the certificate update runs during installation.
 ====
 
+ifdef::azure-private[]
+* If you are deploying a private image registry with your private Microsoft Azure cluster, you can enter the following command to check if a `privateEndpointName` resource has been created for the image registry:
```

Comment:
should we tell users why they would want to check if a `privavteEndpointName` was created?
not sure if relevant, but when the registry fails to create the endpoint a condition in the clusteroperator will indicate that it has failed. At install time I'm pretty sure that the install will be considered failed if the registry operator fails to create the private endpoint (or any other resource for that matter)

Reply:
@flavianmissi How's this? 

Also, is "Cluster Operator" right? Just wondering!

Thanks

```suggestion
* If you have deployed a private image registry with a private Microsoft Azure cluster, you should ensure that a `privateEndpointName` resource was created. If the image registry fails to create the endpoint, the Cluster Operator indicates that installation of the image registry has failed.
```

Reply:
I don't think this is a required step, so maybe using `should` is a bit harsh? I think maybe this can even be removed?
Otherwise yes "Cluster Operator" should be right.

Reply:
Which part are you saying could be removed? the whole thing? Sorry ! 

I can swap should / can and keep the explanation for why they might check it. 

Otherwise I could just put something like:

Optional. Enter the following command to check if a `privateEndpointName` resource has been created for the image registry:

Reply:
@flavianmissi Total removal of this section sounds good to me. @jinyunma What do you think? Should I remove this whole verification step since the installation will fail if an endpoint is not created?

Reply:
yes, if private endpoint fails to be created, registry operator will be degraded and installation will fail.
I'm okay to remove verification step.